<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDialog</class>
 <widget class="QDialog" name="UpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Delete Item</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog, QLabel, QPushButton, QTableWidget{
background-color:rgb(54,54,54);
color:white;
font:12px;
}
QFrame{
border:1px solid grey;
border-radius:10px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit, QLabel{
background-color:rgb(54,54,54);
font:14px;
color:white;
}
QLineEdit#searchLineEdit{
border-radius:3px;
border:1px solid grey;
}
QPushButton#addBtn{
background-color:green;
border:1px solid grey;
color:white;
font:27px;
border-radius:5px;
}
QPushButton#addBtn:hover{
border:2px solid white;
}
QPushButton#addBtn:pressed{
border:3px solid orange;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string>Search :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
        <property name="placeholderText">
         <string>Enter Item Name</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add to table</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>40</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <column>
      <property name="text">
       <string>Product Name</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="background">
       <color>
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>70</red>
         <green>70</green>
         <blue>70</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stock Quantity</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="background">
       <color>
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>70</red>
         <green>70</green>
         <blue>70</blue>
        </color>
       </brush>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit, QLabel{
background-color:rgb(54,54,54);
font:14px;
color:white;
}
QLineEdit#searchLineEdit{
border-radius:3px;
border:1px solid grey;
}
QPushButton{
background-color:green;
border:1px solid grey;
color:white;
font:13px;
border-radius:1px;
}
QPushButton#clearBtn{
background-color: rgb(180, 0, 0);
	font: 75 24pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover{
border:2px solid white;
}
QPushButton:pressed{
border:3px solid orange;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string>Quantity to Add</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(54,54,54);
font:12px;
color:white;
border-radius:5px;
border:1px solid grey;</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateStockTearBtn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add to table</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateStockAddBtn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add to table</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clear Input Fields</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
